pytorch_transformers/tests/modeling_roberta_test.py

```diff
@@ -0,0 +1,69 @@
+# coding=utf-8
+# Copyright 2018 The Google AI Language Team Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import os
+import unittest
+import pytest
+import torch
+
+from pytorch_transformers.modeling_roberta import (RobertaForMaskedLM,
+                                                   RobertaModel)
+
+
+class RobertaModelTest(unittest.TestCase):
```

Comment:
Our model tests derive from `CommonTestCases.CommonModelTester` to share a bunch of common tests (this could be refactored to make it simpler but it works for now).
Maybe the easiest here is to copy/adapt BertModelTest?